nextcloud-startup-script.sh

```diff
@@ -402,7 +403,13 @@ do
             clear
             run_static_script cookielifetime
         ;;
-
+		
+        "Share-Folder")
+            clear
+            occ_command config:system:set share_folder --value="/Shared"
```

Comment:
Don't we need to create the `/Shared` folder, or is it created automatically?

Reply:
It is created automatically. 👍